src/constants.ts

```diff
@@ -39,6 +39,22 @@ export enum GatewayKeyType {
     secondary = "secondary"
 }
 
+export const APIMAccountCommandId = {
+    createAzureAccount: "Create an Azure Account...",
```

Comment:
minor: seems this is not a command id. Can we give it a better name?

Reply:
sorry, mix the command id and string, already updated.